foundations/azure/referenceImplementations/core/managementGroupTemplates/policyDefinitions/Compliant-PurviewPolicySetDefinition.json

```diff
@@ -0,0 +1,213 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-08-01/managementGroupDeploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "topLevelManagementGroupPrefix": {
+            "type": "string",
+            "defaultValue": "FSIDemo"
+        }
+    },
+    "variables": {
+        "scope": "[concat('/providers/Microsoft.Management/managementGroups/', parameters('topLevelManagementGroupPrefix'))]",
+        "policies": {
+            "policyDefinitions": [
+                {
+                    "properties": {
+                        "displayName": "Purview should disable public network acess",
+                        "mode": "Indexed",
+                        "description": "Disabling public network access improves security by ensuring that the Purview is not exposed on the public internet. Creating private endpoints can limit exposure of a Function App. Learn more at: https://learn.microsoft.com/en-us/azure/purview/catalog-private-link",
+                        "metadata": {
```

Comment:
```suggestion
                        "description": "Disabling public network access improves security by ensuring that the Purview is not exposed on the public internet. Creating private endpoints can limit exposure of a Purview.",
                        "metadata": {
```

Reply:
Please review some of the updates for the first policy. There are some improvements that we can make.